polaris/ocean/tests/global_convergence/cosine_bell/viz.py

```diff
@@ -0,0 +1,142 @@
+
+    Attributes
+    ----------
+    mapping_file_step : polaris.remap.MappingFileStep
+        The step for creating a mapping files, also used to remap data
+        from the MPAS mesh to a lon-lat grid
+
+    mesh_name : str
+        The name of the mesh
+    """
+    def __init__(self, test_case, name, subdir, mapping_file_step, mesh_name):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : polaris.TestCase
+            The test case this step belongs to
+
+        name : str
+            The name of the step
+
+        subdir : str
+            The subdirectory in the test case's work directory for the step
+
+        mapping_file_step : polaris.remap.MappingFileStep
+            The step for creating a mapping files, also used to remap data
+            from the MPAS mesh to a lon-lat grid
+
+        mesh_name : str
+            The name of the mesh
+        """
+        super().__init__(test_case=test_case, name=name, subdir=subdir)
+        self.add_input_file(
+            filename='initial_state.nc',
+            target='../init/initial_state.nc')
+        self.add_input_file(
+            filename='output.nc',
+            target='../forward/output.nc')
+        self.mapping_file_step = mapping_file_step
+        self.mesh_name = mesh_name
+        self.add_output_file('init.png')
+        self.add_output_file('final.png')
+
+    def setup(self):
+        """
+        Set up the source and destination grids for this step
+        """
+        super().setup()
+        map_filename = self.mapping_file_step.map_filename
+        self.add_input_file(filename=map_filename,
+                            target=f'../map/{map_filename}')
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        config = self.config
+        mesh_name = self.mesh_name
+        period = config.getfloat('cosine_bell', 'vel_pd')
+
+        remapper = self.mapping_file_step.get_remapper()
+
+        ds_init = xr.open_dataset('initial_state.nc')
+        ds_init = ds_init[['tracer1', ]].isel(Time=0, nVertLevels=0)
+        ds_init = remapper.remap(ds_init)
+
+        plot_global(ds_init.lon.values, ds_init.lat.values,
+                    ds_init.tracer1.values,
+                    out_filename='init.png', config=config,
+                    colormap_section='cosine_bell_viz',
+                    title=f'{mesh_name} tracer at init', plot_land=True)
+
+        ds_out = xr.open_dataset('output.nc')
+        ds_out = ds_out[['tracer1', ]].isel(Time=-1, nVertLevels=0)
+        ds_out = remapper.remap(ds_out)
+
+        plot_global(ds_out.lon.values, ds_out.lat.values,
+                    ds_out.tracer1.values,
+                    out_filename='final.png', config=config,
+                    colormap_section='cosine_bell_viz',
+                    title=f'{mesh_name} tracer after {period} days',
+                    plot_land=True)
```

Comment:
```suggestion
                    plot_land=False)
```
I love that this is an option. Both this test case and a few other Williamson test cases we hope to implement do not have bottom topography.

Reply:
Right, this was mostly to demonstrate the capability.  I agree.